### ISSUE TITLE
The result of my training is very poor，why？

### ISSUE THREAD

**Issue by irisliu10** [1adrianb/binary-face-alignment#11]
Hello,  I tried to train the binary network  in the way of the paper XNOR-Net （https://github.com/allenai/XNOR-Net), however, the training result is very poor, the  structure of my model is the same as the pretrained model you give ,and the training  parameters  are based on the parameters given in the paper. I used BCECriterion, the loss is always around 0.027 when i trained more then 47 epoch, it doesn't drop anymore, and the prediction is poor too. I‘m very confused why the result is so poor ,could you tell me some training details？  Thank you very much!
